src/Definition/ConfigProcessor/ConfigProcessorInterface.php

```diff
@@ -4,9 +4,9 @@
 
 namespace Overblog\GraphQLBundle\Definition\ConfigProcessor;
 
-use Overblog\GraphQLBundle\Definition\LazyConfig;
+use Closure;
 
 interface ConfigProcessorInterface
 {
-    public function process(LazyConfig $lazyConfig): LazyConfig;
+    public function process(array &$config): void;
```

Comment:
The processors don't return the `LazyConfig` anymore, because:

- LazyConfig is removed
- The returned value is never used anyway